project_template/{{cookiecutter.project_repo_name}}/config/config.yaml

```diff
@@ -8,41 +8,18 @@ global:
   env:
     PROJECT_SLUG: '{{cookiecutter.project_slug}}'
 {% raw %}
-    ENVIRONMENT_NAME: {{ env.ENVIRONMENT_NAME }}
     ENVIRONMENT: '{{ env.ENVIRONMENT | default("local", True) }}'
     LOG_LEVEL: INFO
 
-    # Messaging
-    PULSAR_URL: pulsar://pulsar-proxy.pulsar.svc.cluster.local:6650
-    PULSAR_ENDPOINT_URL: http://pulsar-proxy.pulsar.svc.cluster.local
-
-    # Databases
-    MONGO_CLIENT_HOST: mongodb:27017
-    MONGO_CLIENT_USERNAME: root
-
-    POSTGRES_CLIENT_HOST: postgresql
-    POSTGRES_CLIENT_PORT: '5432'
-    POSTGRES_CLIENT_USERNAME: postgres
-
-    REDIS_CLIENT_HOST: redis-master
-    REDIS_CLIENT_PORT: '6379'
-
-    BACKUP_CONTAINER_NAME: '{{ env.BACKUP_CONTAINER_NAME }}'
-    BACKUPS_STORAGE_ACCOUNT_NAME: '{{ env.BACKUPS_STORAGE_ACCOUNT_NAME }}'
-
     # Misc
     DEFAULT_TIME_ZONE: Europe/Zagreb
 
   secrets:
     SENTRY_DSN: '{{ env.SENTRY_DSN }}'
 
-    MONGO_CLIENT_CONNECTION_STRING: '{{ env.MONGO_CLIENT_CONNECTION_STRING }}'
     MONGO_CLIENT_PASSWORD: admin
```

Comment:
@lbenc135 These should be set in CI/CD variables, do they maybe have some defaults for local run?

Reply:
yes, they're all empty by default, which works locally

Reply:
@plavy how do we keep this backwards compatible? does the new CI auto-generate random password?

Reply:
You can always override CI/CD variables with config